geanyvc/src/geanyvc.c

```diff
@@ -1528,7 +1528,12 @@ vccommit_activated(G_GNUC_UNUSED GtkMenuItem * menuitem, G_GNUC_UNUSED gpointer
 	gint height;
 
 #ifdef USE_GTKSPELL
-	GtkSpell *speller = NULL;
+#if !GTK_CHECK_VERSION (3, 0, 0)
+//Since gtk3 will be default in future may be going this way would be better.
+    #define GtkSpellSpellChecker GtkSpell
+    #define gtk_spell_checker_set_language gtkspell_set_language
```

Comment:
might be better to add this compat higher in the file, e.g. near the gtkspell include?